api/definitions/v1.build/description.adoc

```diff
@@ -0,0 +1 @@
+A build is the process of transforming input parameters into a resulting object. Most often, the process is used to transform source code into a runnable image. The OpenShift build system provides extensible support for build strategies that are based on selectable types specified in the build API. There are three build strategies available: link:https://access.redhat.com/beta/documentation/en/openshift-enterprise-30-architecture/chapter-3-core-concepts#docker-build[Docker build], link:https://access.redhat.com/beta/documentation/en/openshift-enterprise-30-architecture/chapter-3-core-concepts#source-build[Source-to-Image (S2I) Build], and link:https://access.redhat.com/beta/documentation/en/openshift-enterprise-30-architecture/chapter-3-core-concepts#custom-build[Custom Build]. A Build encapsulates the inputs needed to produce a new deployable image as a <<v1.BuildSpec>>, as well as the status of the execution and a reference to the Pod which executed the build as a <<v1.BuildStatus>>.
```

Comment:
> A build is the process of transforming input parameters into a resulting object. Most often, the process is used to transform source code into a runnable image.

I'd say a build is the process of transforming source... something like: 
A build is the process of transforming source code into a set of artifacts. Typically, the resulting artifact is a runnable image.

> The OpenShift build system provides extensible support for build strategies that are based on selectable types specified in the build API. There are three build strategies available: 

Seems too wordy. More like:
The OpenShift build system provides support for the following build strategies: